ts/src/pro/binance.ts

```diff
@@ -1314,18 +1314,21 @@ export default class binance extends binanceRest {
             return;
         }
         let method = 'publicPutUserDataStream';
+        const request = {};
+        const symbol = this.safeString (params, 'symbol');
+        const sendParams = this.omit (params, [ 'type', 'symbol' ]);
         if (type === 'future') {
             method = 'fapiPrivatePutListenKey';
```

Comment:
@sc0Vu I'm a bit confused don't we need to send the listening key for linear/inverse perps?
https://binance-docs.github.io/apidocs/futures/en/#keepalive-user-data-stream-user_stream

Reply:
@carlosmiei there is no parameter for linear and inverse market: (https://binance-docs.github.io/apidocs/delivery/en/#keepalive-user-data-stream-user_stream). After test, it really works without listenKey, and won't return error even we sent 1 parameter...... I'm confused too.

Reply:
@sc0Vu ok we can try out your solution